Implementation.md

```diff
@@ -0,0 +1,128 @@
+
+For any given topic, one or two members research independently, then educate the rest of the group on what they have learned. Because some people had prior knowledge (Chris with Github and Brandon with Android Studio), we were able to cover some topics early on during our weekly meetings. Throughout the semester, we have used Slack as an asynchronous resource for posting resources, and keeping everyone up to speed.
+
+Recent events have made in-person meetings difficult, but we have effectively used phone, Discord-based screen share, and slack work to keep us all learning and developing productively. As the app continues to develop, we hope this cycle of discovery, sharing, and adaptation continues.
+
+### 5. Deployment
+
+**Your system should use containers and Docker. Describe how you are deploying your
+system in production. Remember that AWS Educate offers free credits for students. See
+the tutorial at https://docker-curriculum.com/ on how to create a container and deploy
+it on AWS. Provide a link for the system in production.**
+
+Because Firebase runs the services that would normally run on a container, a Docker image is not needed. This highlights another advantage of choosing Firebase for our application and has been approved by Dr. Gerosa.
+
+### 6. Licensing
+
+**Inform the license you adopted for your source code (remember to configure GitHub
+accordingly) and why you adopted this license. For more information, check [https://choosealicense.com/](https://choosealicense.com/) and [http://redhat.slides.com/glikins/open-sourcelicensing-101#/](http://redhat.slides.com/glikins/open-sourcelicensing-101#/)**
+
+**MIT License**
+
+[https://choosealicense.com/licenses/mit/](https://choosealicense.com/licenses/mit/)
+[https://github.com/ChrisKeefe/DontPanic/blob/master/LICENSE](https://github.com/ChrisKeefe/DontPanic/blob/master/LICENSE)
+
+We chose a MIT license so people can do almost anything they want with our project, like making and distributing closed source versions. We want our project to be as accessible as possible for people who want to work on it
+
+### 7. Readme File
+
+**You should also prepare your repository for receiving new contributors. See an example
+of a Readme.md file at [https://gist.github.com/PurpleBooth/109311bb0361f32d87a2](https://gist.github.com/PurpleBooth/109311bb0361f32d87a2)
+Besides the Readme.md file, your repository should contain a CONTRIBUTING.md
+file, a LICENSE file, and a CODE_OF_CONDUCT.md file. Search online for some
+examples of these files. In this section of the deliverable, put links to these files on GitHub.**
+
+[https://github.com/ChrisKeefe/DontPanic/blob/master/README.md](https://github.com/ChrisKeefe/DontPanic/blob/master/README.md)

```

Comment:
These are getting lumped together - can you separate them into separate lines? Maybe a bullet list?